operations/tasks/update-node-selector-labels.yml

```diff
@@ -2,7 +2,7 @@
 # delete all predefined node selector labels which are not needed anymore
 - name: Remove managed node labels
   command: >
-    oc label node {{ inventory_hostname }} {{ item }}-
+    oc label node {{ inventory_hostname }} {{ item | replace('=true', '') }}-
```

Comment:
Is there a way to replace `=` and anything after it? Not sure if "`=.+`" will work for `replace` filter.